src/server/sproutVideo/caption.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+class Captions {
+  constructor(request) {
+    this._request = request;
+  }
+
+  create(videoId, payload) {
+    return this._request.post(`videos/${videoId}/subtitles`, payload);
+  }
+
+  delete(videoId, captionId) {
+    return this._request.delete(`videos/${videoId}/subtitles/${captionId}`);
+  }
+}
+
+module.exports = Captions;
```

Comment:
Does it make sense to use singular nouns for these clients (also to align them with the module names)?